tsdb/segment.go

```diff
@@ -137,6 +145,8 @@ type Segment interface {
 	BaseTime() int64
 	// GetDataFamilyScanners returns data family list by time range, return nil if not match
 	GetDataFamilyScanners(timeRange timeutil.TimeRange) []series.DataFamilyScanner
```

Comment:
are these 2 methods redundant?

Reply:
segment invokes getScanner method for family level scanning